modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -45,7 +45,7 @@ policyModule = function () {
             utility.startTenantFlow(carbonUser);
             var url = mdmProps["httpsURL"] + "/mdm-admin/policies";
             var isUpdated = false;
-            var response = serviceInvokers.XMLHttp.get(url,function(responsePayload){
+            var response = serviceInvokers.XMLHttp.get(url, function (responsePayload) {
```

Comment:
better placing the success callback as a separate function and call it here for readability